test-data/unit/stubgen.test

```diff
@@ -1382,3 +1382,56 @@ from typing import Any, Dict
 
 funcs: Dict[Any, Any]
 f: Any
+
+[case testAbstractMethodNameExpr]
+from abc import ABCMeta, abstractmethod
+
+class A(metaclass=ABCMeta):
+    @abstractmethod
+    def meth(self):
+        pass
+[out]
+from abc import ABCMeta, abstractmethod
+
+class A(metaclass=ABCMeta):
+    @abstractmethod
+    def meth(self) -> None: ...
```

Comment:
Should the return type be `Any`?

Reply:
The return is `Any` only if we run semantic analyzer (test name should end in `_semanal`), see a test below. But on the second thought it looks like it is not always necessary, in most common cases (like when we add the decorator here) it is better to return `Any`. I will fix this now.